app/src/org/commcare/android/database/global/models/AppAvailableToInstall.java

```diff
@@ -0,0 +1,70 @@
+package org.commcare.android.database.global.models;
+
+import org.commcare.android.storage.framework.Persisted;
+import org.commcare.models.framework.Persisting;
+import org.javarosa.core.util.externalizable.DeserializationException;
+import org.javarosa.core.util.externalizable.ExtUtil;
+import org.javarosa.core.util.externalizable.PrototypeFactory;
+
+import java.io.DataInputStream;
+import java.io.DataOutputStream;
+import java.io.IOException;
+
+/**
+ * Represents an app that exists on CommCare HQ (either india or prod) and is accessible for
+ * the currently-authenticated user (either mobile or web) to install.
+ *
+ * Created by amstone326 on 2/3/17.
+ */
+public class AppAvailableToInstall extends Persisted {
+
+    public static final String STORAGE_KEY = "available_apps";
+
+    @Persisting(1)
```

Comment:
I moved it into Android because the `Persisted` class is in Android, and this extends that